frontend/src/metabase/visualizations/components/ChartClickActions.css

```diff
@@ -0,0 +1,34 @@
+.token {
+  padding: 5px 14px 5px 14px;
+  border-radius: 100px;
+  line-height: 16px;
+}
+.token-blue {
+  color: var(--color-brand);
+  border: 1px solid color(var(--color-brand) alpha(-65%));
+}
+.token-filter {
+  padding: 2px 16px;
+  border-radius: 100px;
+  color: var(--color-accent7);
+  border: 1px solid color(var(--color-accent7) alpha(-50%));
+}
+.token-filter:hover {
+  background-color: var(--color-accent7);
+}
+.sort {
+  padding: 4px 6px 4px 12px;
+  border-radius: 100px;
+}
+.sort:hover > svg {
+  color: white;
+}
+.horizontal-button {
+  border-radius: 4px;
+}
+.horizontal-button:hover > svg {
+  color: white;
+}
+.formatting-button {
+  color: color(#b8bbc3 alpha(-50%));
```

Comment:
Is there a reason you're using a color function here as opposed to just having an updated hex value? (Also I think not using a color var here will mean this color won't change if someone sets custom brand colors, although I think that's probably fine in this specific context?)